testing-examples/chatbot-simulation/chatbot-simulation.ipynb

```diff
@@ -362,12 +361,10 @@
     }
    ],
    "source": [
-    "evaluation = RunEvalConfig(evaluators=[did_resist])\n",
-    "\n",
-    "result = client.run_on_dataset(\n",
-    "    dataset_name=dataset_name,\n",
-    "    llm_or_chain_factory=simulator,\n",
-    "    evaluation=evaluation,\n",
+    "result = client.evaluate(\n",
+    "    simulator,\n",
+    "    data=dataset_name,\n",
+    "    evaluators=[did_resist],\n",
```

Comment:
Changed from `run_on_dataset`(deprecated) to `evaluate` method to resolve:
1. Pydantic configuration error
2. Use of deprecated API

For migration guide: https://docs.smith.langchain.com/old/evaluation/migration